commons/src/methods/appendStyles.ts

```diff
@@ -0,0 +1,5 @@
+export default function setShadowStyle(host: any, styleContent: string): void {
+  const style = document.createElement("style");
+  style.innerHTML = styleContent;
+  host.shadowRoot.appendChild(style);
+}
```

Comment:
It would probably be helpful to leave a comment or link to the comment that suggested this workaround. It seems like in order to retain the functionality of svelte components, we have to lose the scoped component styles feature. Since this change seems to be more permanent (I don't see interest from svelte team to fix this in the near future) it'd be nice to hear thoughts from the rest of the team.

Essentially, I think this means:
1) all child svelte component styles need to be in their own file
2) child component styles can now conflict since they're not scoped by the compiler
3) child components lose they're reusability since we have to inject styles into their [specific parent shadow dom](https://github.com/nylas/components/pull/234/files#diff-67d804b56dc33ae8b0a32136a22e93b5acc676fda017034e5fe220f9c4fb786bR4-R7)

cc: @philrenaud @pooja169usp @ozsivanov @AaronDDM